library/src/main/java/com/michaldrabik/classicmaterialtimepicker/model/CmtpDate.kt

```diff
@@ -0,0 +1,22 @@
+package com.michaldrabik.classicmaterialtimepicker.model
+
+import java.util.*
+
+data class CmtpDate(
+  val day: Int,
+  val month: Int,
+  val year: Int
+) {
+
+  companion object {
+    @JvmStatic
+    val calendar: Calendar = Calendar.getInstance()
+    val DEFAULT = CmtpDate(
```

Comment:
I'd rather use some static values here and not the Calendar class. For example 01/01/2000. 
We can second constructor that accepts Calendar instance and creates CmtdDate from it.

Reply:
I don't see much use for a second constructor, since the user could just use the setInitialDate instead... But sure, I'll create the constructor just in case.

Reply:
Ye - actually setInitialDate is enough :)